modules/openapi-generator/src/main/resources/rust/reqwest/api.mustache

```diff
@@ -71,302 +71,307 @@ pub enum {{{operationIdCamelCase}}}Error {
 {{/operation}}
 {{/operations}}
 
+pub struct {{{classname}}};
+
+impl {{{classname}}} {
```

Comment:
This is a preference for sure :) For my part I find code Rust easier to maintain when it is grouped into meaningful structs.